libmscore/layout.cpp

```diff
@@ -1531,6 +1531,31 @@ void Score::connectTies(bool silent)
     }
 }
 
+//---------------------------------------------------------
+//   relayoutToggleStyles
+///   force relayout if certain style settings are enabled
```

Comment:
Somehow this doesn't seem to describe what the function actually appears to do.  This function changes style settings, but it isn't obvious why.

Reply:
How about:

```
///   for styles that can't properly apply to a score that hasn't been laid out yet,
///   temporarily disable them and then reenable after layout
```

Reply:
Added comments that should make it much more clear